src/mapeo-manager.js

```diff
@@ -425,4 +484,10 @@ export class MapeoManager {
   get invite() {
     return this.#invite
   }
+
+  async [kClose]() {
```

Comment:
introduced this because it's needed to properly clean up in tests, since we start the fastify server on init. alternatively could provide an opt within `mediaServerOpts` that can be used in tests to change that behavior for tests, and avoid introducing this.